src/functions-and-arrays.js

```diff
@@ -52,14 +131,41 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let uniqueArr = [];
+  if (arr.length == 0) {
+    return null
+  }
+  for (let i = 0; i < arr.length; i++) {
+    if(uniqueArr.indexOf(arr[i]) == -1){
+       uniqueArr.push(arr[i])
+    }
+    else {
+    continue
```

Comment:
The "else"-condition is not necessary here, because you anyway continue with the next value